docs/adrs/0005-field-methods.md

```diff
@@ -0,0 +1,67 @@
+# 1. Field methods refactor
+
+Date: 2024-05-14
+
+## Status
+
+Proposed
+
+## Context
+
+### Goal
+Simplify the application by refactoring it to use field methods to populate a `TimdexRecord` instance.
+
+### Background
+A `TimdexRecord` contains the following fields: 
+
+* REQUIRED
+    * `source`
+    * `source_link`
+    * `timdex_record_id`
+    * `title`
+
+  * OPTIONAL
+    * `alternate_titles`
+    * `call_numbers`
+    * `citation`
+    * `content_type`
+    * `contents`
+    * `contributors`
+    * `dates`
+    * `edition`
+    * `file_formats`
+    * `format`
+    * `funding_information`
+    * `holdings`
+    * `identifiers`
+    * `languages`
+    * `links`
+    * `literary_form`
+    * `locations`
+    * `notes`
+    * `numbering`
+    * `physical_description`
+    * `publication_frequency`
+    * `publishers`
+    * `related_items`
+    * `rights`
+    * `subjects`
+    * `summary`
+
+Currently much of the transformation logic for each source is found in the `get_optional_fields` method which extracts data for all optional fields. This results in very large methods as well as inefficient orchestration and testing. 
+
+A field method approach would address many of these complications, where each `TimdexRecord` field has a corresponding method containing all of the logic related to extracting and formatting the values for that field. All of the required fields have dedicated field methods as well as some of the complex fields that are called by `get_optional_fields`. This refactor can be seen as an extension of that approach.
```

Comment:
Dig it.  Thanks for explaining what a "field method" is here.

Reply:
I agree! Awesome description.